cost-analyzer/templates/_helpers.tpl

```diff
@@ -945,11 +924,9 @@ Begin Kubecost 2.0 templates
   volumeMounts:
     - name: persistent-configs
       mountPath: /var/configs
-    {{- if or (.Values.kubecostModel).federatedStorageConfigSecret (.Values.kubecostModel).federatedStorageConfig }}
     - name: federated-storage-config
       mountPath: /var/configs/etl
       readOnly: true
-    {{- end }}
```

Comment:
I recommend adding this `if` statement back in? Everywhere else in this PR you are still checking the existance of federatedStorage before using it.

```suggestion
    {{- if or (.Values.global).federatedStorage.existingSecret (.Values.global).federatedStorageConfig }}
    - name: federated-storage-config
      mountPath: /var/configs/etl
      readOnly: true
    {{- end }}
```